parser/mpParserMessageProvider.cpp

```diff
@@ -111,6 +111,8 @@ MUP_NAMESPACE_START
     m_vErrMsg[ecADD_HOURS_DATE]               = _T("The first parameter could not be converted to a date. Please use the format: \"yyyy-mm-dd\"");
     m_vErrMsg[ecADD_HOURS_DATETIME]           = _T("The first parameter could not be converted to a date time. Please use the format: \"yyyy-mm-ddTHH:MM\"");
     m_vErrMsg[ecINVALID_TYPES_MATCH]          = _T("Both values of the default(x, y) function should have the same type");
+    m_vErrMsg[ecUKNOWN_LOCALE]                = _T("The chosen locale is not supported");
+    m_vErrMsg[ecINVALID_TIME_FORMAT]          = _T("Invalid time format on parameter(s). Please use the \"HH:MM:SS\" format.");

```

Comment:
without this error message the program was filing because of it missing, which would stop the tests script from getting the error message. it was not added in the PR that used it

Reply:
I love tests because their incredible ability to highlight these corner cases. :clap: